### PR TITLE
3 changes

### PR DESCRIPTION
**Pull request by matthewma7**
See commit message for detail.

### CODE REVIEW

File: web_external/HeaderBar/style.styl [hunk 1]
```diff
@@ -1,73 +1,24 @@
 .v-header-wrapper
-  display flex
```
Comment (mwoehlke-kitware) [inline web_external/HeaderBar/style.styl:2]: Please do not replace the flex layout with floating elements.

**Comment by matthewma7** [comment]
@mwoehlke-kitware The latest changes are ready to be reviewed. 😃

File: web_external/HeaderBar/style.styl (hunk 1 above)
Comment (matthewma7) [reply web_external/HeaderBar/style.styl:2]: Because?

**Review by mwoehlke-kitware — APPROVED** [review]

**Comment by matthewma7** [comment]
Note: @mwoehlke-kitware suggest use flex, but since float works fine for current needs, flex is not needed for this PR. 
However, considering flex is the modern way to do layout, I will change the usage from float to flex next time when I work on the related feature.

**Pull request merged**